verify/verify_test.go

```diff
@@ -414,3 +417,17 @@ func TestOpenGetExtendedReportVerifyClose(t *testing.T) {
 		}
 	}
 }
+
+func TestRealAttestationVerification(t *testing.T) {
+	var nonce [64]byte
+	copy(nonce[:], []byte{1, 2, 3, 4, 5})
+	getter := &test.Getter{
+		Responses: map[string][]byte{
+			"https://kdsintf.amd.com/vcek/v1/Milan/cert_chain": milanBytes,
+			"https://kdsintf.amd.com/vcek/v1/Milan/3ac3fe21e13fb0990eb28a802e3fb6a29483a6b0753590c951bdd3b8e53786184ca39e359669a2b76a1936776b564ea464cdce40c05f63c9b610c5068b006b5d?blSPL=2&teeSPL=0&snpSPL=5&ucodeSPL=68": vcekBytes,
```

Comment:
Can you add some description for this line? It's not immediately obvious how this URL is generated

Reply:
Done.